src/sentry/apidocs/examples/release_threshold_examples.py

```diff
@@ -0,0 +1,35 @@
+from drf_spectacular.utils import OpenApiExample
+
+from .project_examples import BASE_PROJECT
+
+
+class ReleaseThresholdExamples:
+    THRESHOLD_STATUS_RESPONSE = [
+        OpenApiExample(
+            "Client key with rate limiting",
+            value={
+                f"{BASE_PROJECT['slug']}-v1.0.0": [
+                    {
+                        "project_id": int(BASE_PROJECT["id"]),
```

Comment:
mypy is complaning here and using `int(...)` is not actually converting this to an int on the docs page. Are you sure that this is actually an int? I checked some other APIs and we return ids as strings a lot of the time

Reply:
Yea, pretty sure this should be an int as it's not being serialized at this point

Reply:
I'll just modify this to an actual int

Reply:
oh ok, prob the easiest fix is to hardcode this as some random int then